l10n_co_res_partner/data/ciiu.csv

```diff
@@ -0,0 +1,839 @@
+"id","code","description","division/id","hierarchy","parent/id","section/id","hasDivision","hasParent","hasSection"
+"ciiu.1","SECCIÓN A","AGRICULTURA, GANADERÍA, CAZA, SILVICULTURA Y PESCA",,,,,"False","False","False"
+"ciiu.01",1,"Agricultura, ganadería, caza y actividades de servicios conexas ",,,,"ciiu.1","False","False","True"
+"ciiu.011",11,"Cultivos agrícolas transitorios ","ciiu.01",,,,"True","False","False"
+"ciiu.0111",111,"Cultivo de cereales (excepto arroz), legumbres y semillas oleaginosas ",,,"ciiu.011",,"False","True","False"
+"ciiu.0112",112,"Cultivo de arroz ",,,"ciiu.011",,"False","True","False"
+"ciiu.0113",113,"Cultivo de hortalizas, raíces y tubérculos ",,,"ciiu.011",,"False","True","False"
+"ciiu.0114",114,"Cultivo de tabaco ",,,"ciiu.011",,"False","True","False"
+"ciiu.0115",115,"Cultivo de plantas textiles ",,,"ciiu.011",,"False","True","False"
+"ciiu.0119",119,"Otros cultivos transitorios n.c.p.",,,"ciiu.011",,"False","True","False"
+"ciiu.012",12,"Cultivos agrícolas permanentes ","ciiu.01",,,,"True","False","False"
+"ciiu.0121",121,"Cultivo de frutas tropicales y subtropicales",,,"ciiu.012",,"False","True","False"
+"ciiu.0122",122,"Cultivo de plátano y banano",,,"ciiu.012",,"False","True","False"
+"ciiu.0123",123,"Cultivo de café",,,"ciiu.012",,"False","True","False"
+"ciiu.0124",124,"Cultivo de caña de azúcar",,,"ciiu.012",,"False","True","False"
```

Comment:
@nhomar It is the International Standard Industrial Classification of All Economic Activities, ISIC in English (_Clasificación Industrial Internacional Uniforme de todas las actividades económicas_, or CIIU in Spanish), published by United Nations.

Here some links:

[http://unstats.un.org/unsd/cr/registry/isic-4.asp](url)

[http://unstats.un.org/unsd/cr/registry/regcst.asp?Cl=27&Lg=1](url)

Some variations may apply (e.g. Colombia), depending on each country policy, but normally the standard is implemented as is.

Maybe off-topic, but for this variations, a good idea can be to make a wizzard to import the codes, or choose the standard ones. Anyway, this can be just part of a roadmap for a standard module about ISIC codes.